lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -375,6 +413,23 @@ class LighthouseReportViewer {
       self.opener.postMessage({opened: true}, '*');
     }
   }
+
+  /**
+   * @param {string} url
+   * @param {string[]} categories
+   * @param {string} device
+   */
+  _loadFromPSI(url, categories, device) {
```

Comment:
good news: you can kill off extension-test.js

bad news: we'll need a test for this new viewer functionality.